disputer/index.js

```diff
@@ -51,6 +54,24 @@ async function run(address, shouldPoll, pollingDelay, priceFeedConfig) {
     const gasEstimator = new GasEstimator(Logger);
     const disputer = new Disputer(Logger, empClient, gasEstimator, priceFeed, accounts[0]);
 
+    // The EMP requires approval to transfer the disputer's collateral tokens in order to dispute
+    // a liquidation. We'll set this once to the max value using the Token client.
+    const tokenClient = new TokenBalanceClient(
```

Comment:
nit: do we use this tokenBalanceClient in the disputer anywhere else? If not, would it just be simpler to manually construct the collateralToken instance without using the client?

Reply:
agreed. done